README.md

```diff
@@ -80,6 +80,8 @@ Hydra::Derivatives::Processors::ActiveEncode.timeout = 5.minutes
 
 ```
 
+You can add this to a `config/initializers/` file ([see Scholarsphere as an example](https://github.com/psu-stewardship/scholarsphere/blob/e16586c4bf6f7aed652924acc47dbe4ec774ff00/config/initializers/hydra_derivatives_config.rb#L5)). Another possibility, though unverifiedy, is that you may try adding it to your `config/application.rb` or even vary by environments (e.g. `config/environments/test.rb`).
```

Comment:
Looks great. Maybe tweak unverifiedy...although it does have a nice little Ned Flanders tone...

:shipit: